repository/build.gradle

```diff
@@ -28,15 +28,22 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter-data-jpa") {
         exclude group: 'org.apache.tomcat', module: 'tomcat-jdbc'
     }
-    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '2.0.0'
-    compile "commons-beanutils:commons-beanutils:1.9.3"
-    compile "org.apache.commons:commons-lang3:3.7"
-    compile "io.springfox:springfox-swagger2:${springfoxSwaggerVersion}"
-    compile "io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}"
+    compile (group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '2.0.0') {
+        exclude group: 'com.google.guava', module: 'guava'
+    }
+    compile "commons-beanutils:commons-beanutils:1.9.4"
+    compile "org.apache.commons:commons-lang3:3.11"
+    compile ("io.springfox:springfox-swagger2:${springfoxSwaggerVersion}") {
+        exclude group: 'com.google.guava', module: 'guava'
+    }
+    compile ("io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}") {
+        exclude group: 'com.google.guava', module: 'guava'
```

Comment:
can we add global exclude instead of doing in individual file?

Reply:
We still need one of the version of these package, global exclude would not work.